cloudformation/cloudformation.yaml

```diff
@@ -327,6 +327,16 @@ Resources:
           Id: !Sub trigger_state_machine-${Stage}-4
           Input: "{ \"deliveryDateDaysFromNow\": 4 }"
           RoleArn: !GetAtt [ fulfilmentTriggerRole, Arn ]
+        -
+          Arn: !Ref zuora_fulfilment_salesforce_downloader-[STAGE]
+          Id: !Sub trigger_salesforce_download-${Stage}
+          Inout: "{}"
+          RoleArn: !GetAtt [ FulfilmentWorkersLambdaRole, Arn ]
+        -
+          Arn: !Ref zuora_fulfilment_comparator-[STAGE]
+          Id: !Sub trigger_salesforce_comparator-${Stage}
+          Inout: "{}"
+          RoleArn: !GetAtt [ FulfilmentWorkersLambdaRole, Arn ]
```

Comment:
Does this cloudformation work in CODE? If it doesn't  we might have to merge my pr first that removes the explicit creation of the lambda log group from the template.

Also this will probably mean the comparator only picks up files generated yesterday since the fulfilment generation for today has just started.